module/FinnaConsole/src/FinnaConsole/Controller/UtilController.php

```diff
@@ -26,13 +26,19 @@
  * @link     http://vufind.org/wiki/vufind2:building_a_controller Wiki
  */
 namespace FinnaConsole\Controller;
-use Zend\Config\Config, Zend\Config\Reader\Ini as IniReader,
-    Zend\Console\Console,
-    Zend\Log\Logger, Zend\Log\Writer\Stream,
-    Zend\Stdlib\Parameters,
-    Finna\Search\Solr\Options, Finna\Search\Solr\Params,
-    Finna\Search\UrlQueryHelper,
-    VuFind\Date\Converter as DateConverter;
+
+use Finna\Db\Row\User;
+use Finna\Db\Table\Search;
+use Finna\Search\Solr\Options;
+use Finna\Search\Solr\Params;
+use Finna\Search\UrlQueryHelper;
+use VuFind\Date\Converter as DateConverter;
+use Zend\Config\Config;
+use Zend\Config\Reader\Ini as IniReader;
+use Zend\Db\Sql\Select;
+use Zend\Log\Logger;
+use Zend\Log\Writer\Stream;
+use Zend\Stdlib\Parameters;
```

Comment:
My reasoning for preferring this kind of use-statement list is as follows:

1. It passes current automated tests. Thus, there is no clearly defined rule on how to format use-statement lists.
2. This style is used by PSR-2, which specifies that "There MUST be one use keyword per declaration.". Given that PSR-2 is increasingly adopted as the baseline standard for PHP code, I feel that it is better to follow one clearly defined rule instead of one undefined rule.
3. Some IDEs (like PHPStorm) implement a "import optimization" feature which removes unused use-statements and sorts the existing ones alphabetically. They tend to prefer the PSR-2 style for conformity.
4. At minimum, one use clause per line should be enforced to improve readability (which negates the whitespace gains from using comma).
5. This style improves commit readability and improves the usability of 'git blame'.